src/sentry/notifications/utils/participants.py

```diff
@@ -261,6 +262,21 @@ def disabled_users_from_project(project: Project) -> Mapping[ExternalProviders,
     return output
 
 
+def get_suspect_committers(project: Project, event: Event) -> List[APIUser]:
+    suspect_committers = []
+    committers: Sequence[AuthorCommitsSerialized] = get_serialized_event_file_committers(
+        project, event
+    )
+    user_emails = [committer["author"]["email"] for committer in committers]  # type: ignore
```

Comment:
`make backend-typing` gives the error `error: Value of type "Optional[Any]" is not indexable` for line 270, despite my attempt to type `committers` in line 267, thus I added `# type: ignore`. I also attempted to check its type or check if author and email existed to no avail.

If anyone knows a better fix for this, please let me know :)

Reply:
Thank you, I have that in line 267 but it didn't help either unfortunately